velox/dwio/parquet/writer/Writer.h

```diff
@@ -25,8 +25,12 @@
 #include "velox/dwio/common/WriterFactory.h"
 #include "velox/vector/ComplexVector.h"
 
+#include "velox/dwio/parquet/writer/arrow/Properties.h"
```

Comment:
nit: put the include in the same list above so that clang-format can order them properly